azure-pipelines.yml

```diff
@@ -259,7 +259,6 @@ jobs:
         env:
           VERSION: $(Build.SourceBranch)
           GH_TOKEN: $(GH_TOKEN)
-          CONTACT_FORM_URL: $(CONTACT_FORM_URL)
```

Comment:
Something weird is going on here. Looking at the commit history, I can see two of my commits that are already in master, and this diff doesn't make sense either since this is already what master looks like. 😕

Reply:
hmm, i rebased from master. Let me fix it up.

Reply:
Fixed up the commit history which now doesn't include this file.